invenio_files_rest/helpers.py

```diff
@@ -291,3 +292,13 @@ def create_file(key, path):
                 assert filename.startswith(source)
                 parts = [p for p in filename[len(source):].split(os.sep) if p]
                 yield create_file('/'.join(parts), os.path.join(root, name))
+
+
+def create_file_streaming_redirect_response(obj):
+    """Redirect response generating function."""
+    response = make_response()
+    redirect_url_base = '/secure_files/'
```

Comment:
I would name this, as default, differently, because someone could think that by default it is `insecure` :)
I would also name the function something with Nginx...what about something like `create_response_nginx_xsendfile` or similar?